app/views/staff/events/info.html.haml

```diff
@@ -78,8 +78,8 @@
               %tr
                 %td.text-primary
                   %strong Start date:
-                %td= event.start_date.strftime("%b %d, %Y")
+                %td= event.start_date.to_s(:month_day_year) if event.start_date
```

Comment:
I recommend using try() for this: event.start_date.try(:to_s, :month_day_year)

http://api.rubyonrails.org/v4.2/classes/Object.html#method-i-try